automl/api/Automl.Samples/ListOperationStatus.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START automl_list_operation_status]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.AutoML.V1;
+using Google.LongRunning;
+using System.Collections.Generic;
+
+public class AutoMLListOperationStatus
+{
+    /// <summary>
+    /// Demonstrates using the AutoML client to list operations.
+    /// </summary>
+    /// <param name="projectId">GCP Project ID.</param>
+    /// <param name="location">GCP Project Region.</param>
+    public IEnumerable<Operation> ListOperationStatus(string projectId = "YOUR-PROJECT-ID", string location = "YOUR-PROJECT-LOCATION")
```

Comment:
The description of the location parameter should be adjusted as per my previous comment.